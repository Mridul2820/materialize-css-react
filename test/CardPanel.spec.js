import React from 'react';
import { render } from '@testing-library/react';
import CardPanel from '../src/CardPanel';

describe('<CardPanel />', () => {
  test('renders', () => {
    const { container } = render(
      <CardPanel className="blue-grey">I am a very simple card</CardPanel>
    );

    expect(container).toMatchSnapshot();
  });
});
