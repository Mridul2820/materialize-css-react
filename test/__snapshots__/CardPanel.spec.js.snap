// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardPanel /> renders 1`] = `
<div>
  <div
    class="card-panel blue-grey"
  >
    I am a very simple card
  </div>
</div>
`;
